### DISCUSSION TITLE
Customize Nova Http error page.

### DISCUSSION THREAD

**Discussion by LorenzoAlu** [laravel/nova-issues#4582] [Q&A] [answered]
Hello, everyone. 
I am trying to customize error pages.
In the tools the customization is done by registering Vue components, but it would be nice to be able to customize error pages like Laravel already does.

When you are on a resource page there is no way to customize error page templates.

It would be top of the line to implement this.

**Comment by crynobone** [comment, marked as answer]
You should be able to override `CustomError403`, `CustomError404` and `CustomAppError` components.
